data/governments.txt

```diff
@@ -677,6 +677,20 @@ government "Hai (Unfettered)"
 	"hostile hail" "hostile unfettered"
 	"hostile disabled hail" "hostile disabled unfettered"
 
+government "Hai (Unfettered Challenger)"
+	"display name" "Hai (Unfettered)"
+	swizzle 5
```

Comment:
```suggestion
	swizzle 4
```

Reply:
it's on purpose that it's not the same as the other uhai actually, this is blue iirc, to make it distinct from the other ships